documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -156,73 +144,78 @@ claimRef:
   uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea
 ----
 
-. Deploy the Cluster Operator.
+. Deploy the Cluster Operator:
 +
-[source,shell,subs="+quotes,attributes"]
+[source,shell]
 ----
-kubectl create -f install/cluster-operator -n _my-project_
+kubectl create -f install/cluster-operator -n my-project
 ----
 
-. Recreate your cluster.
-+
-Follow the steps depending on whether or not you have all the `KafkaTopic` resources needed to recreate your cluster.
+. Recreate all `KafkaTopic` resources by applying the `KafkaTopic` resource configuration:
 +
---
-*_Option 1_*: If you have *all* the `KafkaTopic` resources that existed before you lost your cluster, including internal topics such as committed offsets from `__consumer_offsets`:
-
-. Recreate all `KafkaTopic` resources.
-+
-It is essential that you recreate the resources before deploying the cluster, or the Topic Operator will delete the topics.
+[source,shell]
+----
+kubectl apply -f <topic_configuration_file>
+----
 
-. Deploy the Kafka cluster.
-+
-For example:
+. Recreate all `KafkaUser` resources:
+.. If user passwords and certificates need to be retained, recreate the user secrets before recreating the `KafkaUser` resources.
+For example, you can create your own TLS certificate and key file, then create a secret from the file:
 +
-[source,shell,subs="+quotes,attributes"]
+[source,shell]
 ----
-kubectl apply -f _kafka.yaml_
+kubectl create secret generic my-secret \
+--from-file=tls.crt \
+--from-file=tls.key
```

Comment:
This is not as simple. It needs to have the right format etc. The expectation here you be that the user recreates the secrets from some backup / from a dump of the original YAMLs. I would probably just remove the lines 163-169 and left it up to the user.

Maybe you could also add that if the secrets are not recreated, the USer Operator will simply generate new credentials. So it is not that important.